scripts/git-checks.sh

```diff
@@ -52,8 +52,9 @@ done
 # For example, no carriage returns or line feeds should be allowed in filenames.
 export LC_ALL=C
 for rev in $(git rev-list $oldrev..$newrev); do
+	# ignore the developer website directory
 	names=$(git diff-tree $rev -r --root --no-commit-id --name-only \
-		--diff-filter=AR)
+		--diff-filter=AR -- . ':!developers.libra.org')
```

Comment:
@bmwill Are you saying I shouldn't do this in the script directly? I think it makes sense to support filenames that the website needs even if they don't align with this particular check.

Reply:
Yes, please do not modify this script. If you want to disable whitespace checks done by this script then take a look at adding a .gitattributes files like I linked above.